zerver/webhooks/stripe/view.py

```diff
@@ -150,7 +155,8 @@ def default_body(update_blacklist: List[str]=[]) -> str:
         if event == 'upcoming':  # nocoverage
             body = 'Upcoming invoice created'
         else:
-            body = default_body(update_blacklist=['lines', 'description', 'number', 'finalized_at'])
+            body = default_body(
+                update_blacklist=['lines', 'description', 'number', 'finalized_at'])
         if event == 'created':  # nocoverage
```

Comment:
The above changes that break function calls over multiple lines are unnecessary, please reverse them!